keps/0001-kubernetes-enhancement-proposal-process.md

```diff
@@ -4,8 +4,6 @@ authors:
   - "@calebamiles"
   - "@jbeda"
 owning-sig: sig-architecture
-participating-sigs:
-  - kubernetes-wide
```

Comment:
At some point, we probably want to reintroduce this idea?
Maybe something like `scope: group|project`?

Reply:
yes we should